### PR TITLE
Development

### PR DESCRIPTION
**Pull request by sijanstu**
Replacing Bot Logo in Description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <p align="center">
-    <a href="https://www.askbuddie.com">
-        <img src="https://raw.githubusercontent.com/askbuddie/readme/master/ask-buddie-icon-200x150.png" align="center" alt="askbuddie-icon"/>
+    <img src="asklogo-noback.png" width="300">
```
Comment (ashiishme) [inline README.md:2]: @sijanstu Can you update the image with the white background? Also, please use this file name: `askbuddie-bot.png`

**Review by ashiishme — CHANGES_REQUESTED** [review]

**Comment by sijanstu** [comment]
Renamed and replaced Background color

File: README.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <p align="center">
-    <a href="https://www.askbuddie.com">
-        <img src="https://raw.githubusercontent.com/askbuddie/readme/master/ask-buddie-icon-200x150.png" align="center" alt="askbuddie-icon"/>
+    <img src="askbuddie-bot.png" width="300">
```
Comment (sijanstu) [inline README.md:2]: Renamed and changed background color of logo

**Review by sijanstu — COMMENTED** [review]
Changed logo with white background: 
> askbuddie-bot.png

File: README.md (hunk 2 above)
Comment (ashiishme) [reply README.md:2]: It's missing the `alt` attribute.

Comment (sijanstu) [reply README.md]: Image is not displaying when I add alt attribute, how to fix it?

Comment (sijanstu) [reply README.md]: image

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <p align="center">
-    <img src="askbuddie-bot.png" width="300">
-    </a>
+     <a href="https://www.askbuddie.com">
+         <img "askbuddie-bot.png" align="center" alt="askbuddie-icon" width="300"/>
```
Comment (ashiishme) [inline README.md:3]: You are missing `src` attribute.

`<img "askbuddie-bot.png"` should be `<img src="askbuddie-bot.png"`.

**Review by ashiishme — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sijanstu) [reply README.md]: Updated to 
`<img src="askbuddie-bot.png"`

**Review by ashiishme — APPROVED** [review]

**Pull request merged**